testing/regress/ecl/javaembed_ex7.manifest

```diff
@@ -1,3 +1,3 @@
 <Manifest>
- <Resource type='jar' filename='javaembed_ex7.jar'/>
+ <Resource type='jar' filename='javaembed_ex7.jar' repository='/generic-local/javaFiles/' md5='d933045e7fed10ff9ccddfc9743b0538'/>
```

Comment:
I assume this is for testing and it should be reverted before merging.

Reply:
Yes, that will be reverted.